package.json

```diff
@@ -96,10 +98,19 @@
   "babel": {
     "presets": [
       "react",
-      "es2015"
+      ["env", {
+        "targets": {
+          "node": "current",
+          "browsers": "defaults"
+        }
+      }],
+      "flow"
     ],
     "plugins": [
-      "transform-flow-strip-types"
+      ["transform-runtime", {
+        "polyfill": false,
+        "regenerator": true
+      }]
```

Comment:
yep we'd still be cross browser; "defaults" means http://browserl.ist/?q=defaults (I added this in a comment but it got hidden after some changes, I think :) )